basic/src/test/java/org/jboss/hal/testsuite/test/runtime/SimpleWebserviceEndpointTestCase.java

```diff
@@ -113,7 +121,7 @@ public void after() throws Exception {
     @AfterClass
     public static void afterClass() throws IOException, TimeoutException, InterruptedException {
         try {
-            operations.undefineAttribute(WEBSERVICES_ADDRESS, "statistics-enabled");
+            operations.writeAttribute(WEBSERVICES_ADDRESS, STATISTICS_ENABLED, originalStatisticsEnabledValue);
```

Comment:
I believe you should be handling here when you have not original value. This is more probable than the backup/restore case as actual reading of the attribute may fail. And instead of seeing error that it failed to connect it could become hidden.